lib/at_host/at_host.c

```diff
@@ -67,16 +68,41 @@ static void response_handler(const char *response)
 	write_uart_string(response);
 }
 
+#if defined(CONFIG_AT_HOST_CUSTOM_COMMANDS)
+static bool cmd_is_custom(const char *cmd)
+{
+	char *prefix = CONFIG_AT_HOST_CUSTOM_COMMANDS_PREFIX;
+
+	for (int i = 0; i < strlen(prefix); i++) {
+		if (tolower(prefix[i]) != tolower(cmd[i])) {
+			return false;
+		}
+	}
+
+	return true;
+}
+#endif /* CONFIG_AT_HOST_CUSTOM_COMMANDS */
+
 static void cmd_send(struct k_work *work)
 {
 	int               err;
+	bool              custom = false;
 
 	ARG_UNUSED(work);
 
-    /* Sending through string format rather than raw buffer in case
-     * the buffer contains characters that need to be escaped
-     */
-	err = nrf_modem_at_cmd(at_buf, sizeof(at_buf), "%s", at_buf);
+#if defined(CONFIG_AT_HOST_CUSTOM_COMMANDS)
```

Comment:
I'd rather see this as a C-style check, so thtat the code is compiled by the CI, i. e. `if (IS_ENABLED(CONFIG_AT_HOST_CUSTOM_COMMANDS)) {`